package.json

```diff
@@ -52,6 +52,7 @@
         "babel-preset-nano-react-app": "^0.1.0",
         "chart.js": "^2.9.3",
         "chartjs-adapter-dayjs": "^1.0.0",
+        "chartjs-plugin-crosshair": "^1.1.6",
```

Comment:
Context on this @samwinslow ? We might no longer need it in this PR right? In general I'm thinking of getting this PR to a usable state and merge already as it's quite large as it is.

Reply:
This renders the vertical scrubber line — without it, you need to hover each graph series individually and won't easily be able to see multiple series in the same tooltip.

Reply:
But you're moving us to D3 anyways?

Reply:
Yep! Can probably remove this for now and use whatever you'd like from the React components created in this branch.